lib/routes/chainswap/send/send_chainswap_confirmation_page.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:breez_liquid/breez_liquid.dart';
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:l_breez/bloc/account/account_bloc.dart';
+import 'package:l_breez/bloc/chainswap/chainswap_bloc.dart';
+import 'package:l_breez/routes/chainswap/send/fee/fee_chooser/fee_chooser.dart';
+import 'package:l_breez/routes/chainswap/send/fee/fee_option.dart';
+import 'package:l_breez/routes/chainswap/send/send_chainswap_button.dart';
+import 'package:l_breez/widgets/loader.dart';
+
+class SendChainSwapConfirmationPage extends StatefulWidget {
+  final int amountSat;
+  final String onchainRecipientAddress;
+  final bool isMaxValue;
+
+  const SendChainSwapConfirmationPage({
+    super.key,
+    required this.amountSat,
+    required this.onchainRecipientAddress,
+    required this.isMaxValue,
+  });
+
+  @override
+  State<SendChainSwapConfirmationPage> createState() => _SendChainSwapConfirmationPageState();
+}
+
+class _SendChainSwapConfirmationPageState extends State<SendChainSwapConfirmationPage> {
+  bool isAffordable = false;
+  PreparePayOnchainResponse? feeOption;
+
+  late Future<PreparePayOnchainResponse> _preparePayOnchainResponseFuture;
+
+  @override
+  void initState() {
+    super.initState();
+    _preparePayOnchainResponse();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final texts = context.texts();
+
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(texts.csv_exporter_fee),
+      ),
+      body: FutureBuilder(
+        future: _preparePayOnchainResponseFuture,
+        builder: (context, snapshot) {
+          if (snapshot.error != null) {
+            return _ErrorMessage(
+              message: (snapshot.error is PaymentError_InsufficientFunds)
+                  ? texts.reverse_swap_confirmation_error_funds_fee
+                  : texts.reverse_swap_confirmation_error_fetch_fee,
+            );
+          }
+          if (snapshot.connectionState != ConnectionState.done) {
+            return const Center(child: Loader());
+          }
+
+          if (isAffordable) {
+            return FeeChooser(feeOption: snapshot.data!);
```

Comment:
I think I need to do another pass to see what `FeeChooser` is doing. Is it just fee confirmation?

Reply:
Yes, it's just fee confirmation.

It was displaying fee options for economy, regular & priority fee rates on c-breez. It displays fee breakdown of `OnchainPaymentLimitsResponse` on l-breez.

It's not really needed, we can use FeeBreakdown directly. 

This whole UI code will change with #6, that's why it's in a poor state.

Reply:
@erdemyerebasmaz perhaps hold off on this. We are going to add the option for the user to select the fee rate or use the boltz default claim estimated fee.

Reply:
> @erdemyerebasmaz perhaps hold off on this. We are going to add the option for the user to select the fee rate or use the boltz default claim estimated fee.

@dangeross Holding off these payment UI PR's is a blocker for new UI changes on #6.
Let's add it once it's implemented on top of new UI 👍